jso/apis/src/main/java/org/teavm/jso/dom/events/WheelEvent.java

```diff
@@ -0,0 +1,35 @@
+/*
+ *  Copyright 2014 Alexey Andreev.
```

Comment:
I usually put the year when the file was actually created. I don't know whether I should update it.